src/language/typescript.ts

```diff
@@ -671,21 +673,65 @@ const client = `
 		} 
 	
 		constructor(readonly errors: Errors) {
-			super('ResponseValidationError');
+			super(report(left(errors)).toString());
+			this.name = 'ResponseValidationError';
 			Object.setPrototypeOf(this, ResponseValidationError);
 		}
 	}
 `;
 
 const utils = `
-	import { Type, success, identity } from 'io-ts';
+	import { Type, success, identity, Errors, Validation, Context, ValidationError } from 'io-ts';
+
 	export const unknownType = new class UnknownType extends Type<unknown> {
 		readonly _tag: 'UnknownType' = 'UnknownType';
-	
+
 		constructor() {
 			super('unknownType', (_: unknown): _ is unknown => true, success, identity);
 		}
 	}();
+
+	function getMessage(e: ValidationError) {
+		return e.message !== undefined ? e.message : 
+			createMessage(e.context)
+			+ '\\n in context: \\n' 
```

Comment:
please uses string templates instead of concatenation and `\\n`

Reply:
that would be really inconvenient, it means losing control of the output, cause rendered indentation will depend on source code indentation and it varies